game/src/plugins/entity/spawn/src/spawn.kt

```diff
@@ -13,3 +13,7 @@ object Spawns {
 fun npc_spawn(name: String, x: Int, y: Int, id: Int? = null) {
```

Comment:
Remove this method - add default height of in your newly added ```npc_spawn``` method.

Also consider adding Direction as a parameter and setting its default to NORTH